package.json

```diff
@@ -18,12 +18,12 @@
     "prepublish": "./node_modules/.bin/grunt build"
   },
   "dependencies": {
-    "can": "^2.2.5",
     "cssify": "0.6.0",
     "d3": "<=3.5.0",
     "c3": "0.4.10"
   },
   "devDependencies": {
+    "can": "^2.2.5",
```

Comment:
I’ve created a new issue for fixing the dependencies: https://github.com/bitovi-components/bit-c3/issues/27